announcement/announcement-tool/tool/src/java/org/sakaiproject/announcement/tool/AnnouncementAction.java

```diff
@@ -2967,6 +2990,13 @@ public void doDelete(RunData rundata, Context context)
 					//AnnouncementMessageEdit edit = channel.editAnnouncementMessage(message.getId());
 					//channel.removeMessage(edit); 
 					channel.removeAnnouncementMessage(message.getId());
+
+					//SAK-44622
```

Comment:
I'd remove the SAK number, we can get that via git history.

Reply:
done